### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy-soar/issues/29#issuecomment-1128193327

Direct copy of the changelog from the release page.

### CODE REVIEW

**Review by dstansby — CHANGES_REQUESTED** [review]
I can edit this myself later today if that's easier, but it would be nice to stick to the current style and remove the dev-only changes (tests/pre-commit) and add a short example for the path string interpolation (I don't think that phrase will mean much to users)

**Comment by nabobalis** [comment]
> I can edit this myself later today if that's easier, but it would be nice to stick to the current style and remove the dev-only changes (tests/pre-commit) and add a short example for the path string interpolation (I don't think that phrase will mean much to users)

Up to you, I can have a go now.

File: README.rst [hunk 1]
```diff
@@ -84,6 +84,13 @@ Contributions for new features and bug fixes are welcome.
 Changelog
 =========
 
+1.3
+---
+
+- Support path string interpolation by @wtbarnes in https://github.com/sunpy/sunpy-soar/pull/24
+  This allows one to do the following: Fido.fetch(query, path=tmp_path / '{instrument}') and the name of the intrument will be used in the save path.
+  This will work for all supported Fido attrs.
```
Comment (nabobalis) [inline README.rst:92]: I don't actually know if this is true.

**Review by dstansby — APPROVED** [review]

**Comment by dstansby** [comment]
Thanks a lot!

**Pull request merged**